Tests/TensorFlowTests/TensorAutoDiffTests.swift

```diff
@@ -782,6 +782,196 @@ final class TensorAutoDiffTests: XCTestCase {
                                 in: productSqueezingAxes01),
                        [[[0, 8640], [0, 5760], [0, 4320]], [[0, 6912], [1575, 5760], [0, 5760]]])
     }
+    
+    func testResizeNearest() {
```

Comment:
I'm not really sure, if we should put this is `TensorAutoDiffTests.swift`, similar to linear algebra operator testing, putting these gradient tests under `ResizeTests.swift`, might result into the same test suite containing everything and easier to locate. WDYT?

Reply:
I thought all gradient tests should go into TensorAutoDiffTests.swift.
Is there consensus where to put them?